scripts/setup-gcloud-cluster.sh

```diff
@@ -62,10 +62,10 @@ usage() {
 : "${FAUCET_SECRET_API_KEY?"Missing environment variable FAUCET_SECRET_API_KEY"}"
 : "${DEPLOYER_PRIVATE_KEY?"Missing environment variable DEPLOYER_PRIVATE_KEY"}"
 
-declare environment="${1?"missing parameter <environment>"}"
+declare network="${1?"missing parameter <network>"}"
 declare init_script=${2:-}
-declare cluster_id="${3:-${environment}-topology-${RANDOM}-${RANDOM}}"
-declare docker_image=${4:-gcr.io/hoprassociation/hoprd:${environment}}
+declare cluster_id="${3:-${network}-topology-${RANDOM}-${RANDOM}}"
+declare docker_image=${4:-gcr.io/hoprassociation/hoprd:${network}}
```

Comment:
There is an error in the default value for `docker_image`. There is not such kind of tag, nevertheless is not reproducable because the 4th argument is always provided.
Better not to change anything at this point, as this will be soon deprecated.

Reply:
We can avoid this change, as mentioned is not impacting at this moment.